codelabs/context_menu_option/context_menu_option.md

```diff
@@ -164,9 +164,11 @@ You can now test `scope.focusedNode` to display the "Hello World" option in work
   };
 ```
 
+Notice that the code tests for where context menus are allowed, rather than where they are not allowed. This is because custom code (such as a plugin) can add context menus to any Blockly component that can be focused. Thus, testing for what something isn't may result in allowing context menus on more components than you anticipated.
```

Comment:
> Thus, testing for what something isn't

How about... Thus, testing for specific types rather than allowing all (or all but certain types) will ensure that context menus are not shown on more components than you anticipated.

I'm not sure if that's actually better. If you don't like that, maybe putting `isn't` in italics would help make it read the appropriate emphasis?